frontend/src/components/widgets/PieChartItem.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <VRow class="justify-center">
+        <VCol class="d-flex col-5 justify-center align-items-center">
+            <GChart
+                class="pb-2 align-self-end"
+                type="PieChart"
+                :data="chartData"
+                :options="chartOptions"
+            />
+        </VCol>
+        <VCol>
+            <VSubheader
+                v-text="dataType"
+                class="legend-title grey--text text--darken-1 pl-0"
+            />
+            <VList>
+                <LegendItem
+                    v-for="(item, key) in data"
+                    :key="key"
+                    :color="item.color"
+                    :title="item.title"
+                    :percent="item.percent"
+                />
+            </VList>
+        </VCol>
+    </VRow>
+</template>
+
+<script>
+    import { GChart } from "vue-google-charts";
+    import LegendItem from "@/components/widgets/LegendItem";
+    export default {
+        name: "PieChartItem",
+        components: { GChart, LegendItem },
+        props: {
+            data: {
+                type: Array,
+                required: true
+            },
+            dataType: {
+                type: String,
+                required: true
+            }
+        },
+        computed: {
+            chartOptions() {
+                return {
+                    chartArea: {
+                        backgroundColor: 'transparent',
+                        width: '85%',
+                        height: '85%'
+                    },
+                    width: 110,
+                    height: 110,
+                    pieHole: 0.87,
+                    pieSliceBorderColor: 'none',
+                    legend: 'none',
+                    pieSliceText: 'none',
+                    tooltip: {
+                        trigger: 'hover'
+                    },
+                    slices: Object.assign({}, this.data.map(item => {
```

Comment:
why you need Object.assign here?

also it's probably better make own computed prop for slices, just to keep it simpler